toolkit/themes/shared/design-system/config.js

```diff
@@ -37,6 +37,8 @@ const MEDIA_QUERY_PROPERTY_MAP = {
   "prefers-contrast": "prefersContrast",
 };
 
+const BASE_SELECTOR = ":root,\n" + ":host(.anonymous-content-host) {\n";
```

Comment:
Moving this to a template string could be a smaller change that helps clarity a bit, but not too concerned

```suggestion
const BASE_SELECTOR = `:root,
:host(.anonymous-content-host) {
`;
```